src/components/TopicDetails/RenderTopicDetails.js

```diff
@@ -41,20 +41,23 @@ function RenderTopicDetails(props) {
   return (
     <div className="topicDetails__container">
       {topicDetailsInfo ? (
-        <div>
+        <div className="innerTopicDetails">
           <h2>{topicDetailsInfo.title}</h2>
+          <Dropdown overlay={menu}>
+            <Button onClick={e => e.preventDefault()}>Select</Button>
+          </Dropdown>
           <h2>Members: </h2>
-          {topicDetailsInfo.members.map(member => (
-            <span>{`${member.name}, `}</span>
-          ))}
+          {topicDetailsInfo.members.map(member => {
+            console.log(member);
```

Comment:
Please remove any console.logs from main. Print statements don't have a place in production code.